metadata/src/main/java/edu/unc/lib/dl/util/SoftwareAgentConstants.java

```diff
@@ -0,0 +1,41 @@
+package edu.unc.lib.dl.util;
+
+public class SoftwareAgentConstants {
+	private static String cdrVersion;
+	
+	public static void setCdrVersion(String cdrVersion) {
+		SoftwareAgentConstants.cdrVersion = cdrVersion;
+	}
+	
+	public static String getCdrVersion() {
+		return cdrVersion;
+	}
+
+	public enum SoftwareAgent {
+		depositService("deposit", cdrVersion), 
```

Comment:
Passing in cdrVersion here doesn't do anything since it is null at that point, I'd suggest switching to depositService("deposit") and overloading the constructor to avoid confusion